index.bs

```diff
@@ -216,6 +216,7 @@ using an [[RDF]] [vocabulary](shapetree.ttl).
 A <dfn>shape tree instance</dfn> is a resource or set of resources assigned to 
 and in conformance with a given [=shape tree=]. A resource in a
 [=shape tree instance=] is called a <dfn>managed resource</dfn>.
+A resource can become a [=managed resource=] with the [=Plant Operation=].
```

Comment:
Agree with this add but I think it would work better at the end of the next paragraph, after shape tree locators are introduced (since plant essentially sets those)